test-one/src/index.js

```diff
@@ -2,13 +2,35 @@ import { log } from "./helpers/logger";
 
 export const go = async () => {
   // Your code should go here instead of the welcome message below. Create and modify additional files as necessary/appropriate.
+  var message = "Result-Above";
 
-  const welcomeMessage =
-    "Congrats, you're up and running! Welcome to the first moshtix code challenge! " +
-    "Please replace this message with your solution and feel free to add " +
-    "or modify other files as necessary. Good luck : )";
+  var test = "";
 
-  await log(welcomeMessage);
+  var i;
+  for(i = 1; i < 101; i++){
```

Comment:
Can you think of any reasons we could benefit from separating this logic into it's own function?